app/src/main/java/edu/usf/sas/pal/muser/model/UiEventType.java

```diff
@@ -15,9 +15,11 @@
          REPEAT_OFF,
          REPEAT_ALL_SONGS,
          REPEAT_CURRENT_SONG,
+         SCAN_FORWARD,
```

Comment:
Could you go ahead and add SCAN_BACKWARD here as well in the same PR? That way all the related events are together.